### PR TITLE
Add references between neighborhood and group tables

### PR DESCRIPTION
**Pull request by HacksForHugs**
Added the group foreign key in neighborhood table and vice versa
Added indexes to both

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -67,6 +74,7 @@
     t.date     "date_of_birth"
     t.string   "phone"
     t.string   "cell_phone"
+    t.string   "neighborhood"
```
Comment (hidesminimally) [inline db/schema.rb]: @HacksForHugs Why do users have a neighborhood string attribute? 
Remember that neighborhoods are related to users through the userNeighborhood model. 

Also, change your github so you have your name on it please lol

File: db/schema.rb [hunk 2]
```diff
@@ -11,25 +11,32 @@
...
 
+  add_index "groups", ["neighborhood_id"], name: "index_groups_on_neighborhood_id", using: :btree
+
   create_table "neighborhoods", force: :cascade do |t|
     t.string   "title"
     t.text     "text"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.string   "location"
+    t.string   "user"
```
Comment (hidesminimally) [inline db/schema.rb]: @HacksForHugs This shouldn't be here. This should be in the neighborhood to user join table.

File: db/migrate/20151020045115_remove_user_from_neighborhood.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+class RemoveUserFromNeighborhood < ActiveRecord::Migration
+  def change
```
Comment (hidesminimally) [inline db/migrate/20151020045115_remove_user_from_neighborhood.rb]: @HacksForHugs If neighbors don't have user attributes, this doesn't seem relevant

File: db/migrate/20151020045234_remove_neighborhood_from_user.rb [hunk 4]
```diff
@@ -0,0 +1,5 @@
+class RemoveNeighborhoodFromUser < ActiveRecord::Migration
+  def change
+  	remove_column :users, :neighborhood
```
Comment (hidesminimally) [inline db/migrate/20151020045234_remove_neighborhood_from_user.rb]: @HacksForHugs Same here

File: db/migrate/20151020033930_add_relation_between_user_and_neighborhood.rb [hunk 5]
```diff
@@ -0,0 +1,9 @@
+class AddRelationBetweenUserAndNeighborhood < ActiveRecord::Migration
+  def change
```
Comment (hidesminimally) [inline db/migrate/20151020033930_add_relation_between_user_and_neighborhood.rb]: @HacksForHugs This isn't adding relations between user and neighborhood. It seems to be adding relations between group and neighborhood. Rename?

**Comment by HacksForHugs** [comment]
@lilshim removed the four files!

**Comment by HacksForHugs** [comment]
@lilshim renamed the migration

**Comment by hidesminimally** [comment]
@HacksForHugs Looks good to me!

**Pull request merged**